package.json

```diff
@@ -22,6 +22,6 @@
     "sha3": "^1.1.0"
   },
   "browser": {
-    "sha3": "browserify-sha3"
+    "index.js": "browser.js"
```

Comment:
Meteor currently supports `"browser": "browser.js"`, which does the same thing you're doing here.

Thanks for being willing to adapt your code to make things easier for Meteor!